Makefile

```diff
@@ -117,28 +117,24 @@ docker-clean docker-clean-enterprise-gateway docker-clean-nb2kg docker-clean-yar
 # itest should have these targets up to date: bdist kernelspecs docker-image-enterprise-gateway 
 
 # itest configurable settings
-# indicates which host (gateway or notebook) to connect to...
+# indicates which host (gateway) to connect to...
 ITEST_HOST:=localhost:8888
```

Comment:
You probably want to enable overriding these by using `ITEST_HOST?=localhost:8888`, this is true for at least `ITEST_HOST`, `ITEST_USER`, `PREP_DOCKER`, you probably have a better idea if there are use cases where the other configurations should be also overridable.

Reply:
I agree with the idea to override the variables.  I do find, however, that `:=` still honors command-line overrides.  That said, I don't see that behavior documented anywhere while `?=` is documented to do the same - so we should probably make that change.